language/Portuguese_Brazil/strings.po

```diff
@@ -687,19 +692,19 @@ msgstr "Automático"
 
 #: src/frontend/mame/ui/menu.cpp:1178
 msgid "Start Machine"
-msgstr ""
+msgstr "Máquina Inicial"
```

Comment:
I’d like to confirm that this is translated correctly.  This is an imperative – selecting the option causes the machine to be started.  It is not a label for the machine to launch on starting.

Reply:
The same answer as above, thanks for telling me, i'll fix them.

Reply:
Fixed and done.

Reply:
Thanks, I think this can be merged now.  If Portuguese-speakers notice any other issues, there’s plenty of time to fix them before the next release.